### PR TITLE
Create sl.js

### PR DESCRIPTION
**Pull request by Morgosh**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [ ] It's submitted to the `dev` branch and _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] It's been tested with all Quasar themes
- [ ] It's been tested on a Cordova (iOS, Android) app
- [ ] It's been tested on a Electron app
- [ ] Any necessary documentation has been added or updated [in the docs](https://github.com/quasarframework/quasar-framework.org/tree/master/source) (for faster update click on "Suggest an edit on GitHub" at bottom of page) or explained in the PR's description.

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

**Comment by Morgosh** [comment]
Added Slovenian language

File: i18n/sl.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    days: 'Nedelja_Ponedeljek_Torek_Sreda_Četrtek_Petek_Sobota'.split('_'),
+    daysShort: 'Ned_Pon_Tor_Sre_Čet_Pet_Sob'.split('_'),
+    months: 'Januar_Februar_Marec_April_Maj_Junij_Julij_August_September_October_November_December'.split('_'),
+    monthsShort: 'Jan_Feb_Mar_Apr_Maj_Jun_Jul_Aug_Sep_Oct_Nov_Dec'.split('_'),
+    firstDayOfWeek: 1, // 0-6, 0 - Nedelja, 1 Ponedeljek, ...
+    format24h: true
+  },
+  pullToRefresh: {
+    pull: 'Za ponovno nalaganje povleci navzdol',
+    release: 'Za ponovno nalaganje pusti',
```
Comment (borutjures) [inline i18n/sl.js]: First: Thank you Morgosh for translations! One less thing on my tasklist.
Suggestion: "pusti" => "spusti"

File: i18n/sl.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    refresh: 'Ponovno nalogam...'
```
Comment (borutjures) [inline i18n/sl.js]: Typo: "nalogam" => "nalagam"

File: i18n/sl.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  },
+  table: {
+    noData: 'Ni dosegljivih podatkov',
+    noResults: 'Ne najdem ustreznic',
+    loading: 'Nalagam...',
+    selectedRows: function (rows) {
+      return rows === 1
+        ? '1 selected row.'
+        : (rows === 0 ? 'No' : rows) + ' selected rows.'
+    },
```
Comment (borutjures) [inline i18n/sl.js]: Not translated.
"1 selected row." => "1 izbrana vrstica."
(rows === 0 ? 'No' : rows) + ' selected rows.'  =>  (rows === 0 ? 'Ni' : rows) + ' izbranih vrstic.'

To also account for Slovenian special case plural:
      return rows === 1
        ? '1 izbrana vrstica.'
        : (rows === 2 ? '2 izbrani vrstici.' : (rows === 0 ? 'Ni' : rows) + ' izbranih vrstic.')

File: i18n/sl.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    bold: 'Krepko',
+    italic: 'Ležeče',
+    strikethrough: 'Prečrtano',
+    underline: 'Podčrtaj',
+    unorderedList: 'Neoštevilčen seznam',
+    orderedList: 'Oštevilčen seznam',
+    subscript: 'Podpisano',
+    superscript: 'Nadpisano',
+    hyperlink: 'Hiper povezava',
+    toggleFullscreen: 'Toggle Fullscreen',
```
Comment (borutjures) [inline i18n/sl.js]: Not translated.
"Toggle Fullscreen" => "Preklopi celoten zaslon"
Sorry, I don't have better/shorter translation for this.

**Comment by rstoenescu** [comment]
Made requested changes by @borutjures .
Thank you both!! Will be available in 0.15.6.

**Pull request merged**